pset-functions.js

```diff
@@ -136,6 +171,86 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+function scrabbleScore(word) {
+  const letters = word.split('')
+  let counter = 0
+  for (let i = 0; i <= word.length; i++) {
+    if (letters[i] === 'a' || letters[i] === 'e' || letters[i] === 'i' || letters[i] === 'o' || letters[i] === 'u' || letters[i] === 'l' || letters[i] === 'n' || letters[i] === 'r' || letters[i] === 's' || letters[i] === 't') {
+    //console.log(1)
+    counter += 1
+    //console.log(counter)
+  } else if (letters[i] === 'd' || letters[i] === 'g') {
+    //console.log(2)
+    counter += 2
+    //console.log(counter)
+  } else if (letters[i] === 'b' || letters[i] === 'c' || letters[i] === 'm' || letters[i] === 'p') {
+    //console.log(3)
+    counter += 3
+    //console.log(counter)
+  } else if (letters[i] === 'f' || letters[i] === 'h' || letters[i] === 'v' || letters[i] === 'w' || letters[i] === 'y') {
+    //console.log(4)
+    counter += 4
+    //console.log(counter)
+  } else if (letters[i] === 'k') {
+    //console.log(5)
+    counter += 5
+    //console.log(counter)
+  } else if (letters[i] === 'j' || letters[i] === 'x') {
+    //console.log(8)
+    counter += 8
+    //console.log(counter)
+  } else if (letters[i] === 'q' || letters[i] === 'z') {
+    //console.log(10)
+    counter += 10
+    //console.log(counter)
+  }
+}
+console.log(`counter is ${counter}`)
+}
+
+scrabbleScore('javascript')
+//console.log(answer)
+
+//assigns scores to letters
+/*const letters = {
```

Comment:
@anniecdavis27 I see where you were going here, that's also another route. great job thinking through alternative approaches